fusestore/store/src/tests/service.rs

```diff
@@ -94,6 +94,9 @@ pub fn new_test_context() -> StoreTestContext {
     let t = tempdir().expect("create temp dir to store meta");
     config.meta_dir = t.path().to_str().unwrap().to_string();
 
+    let t = tempdir().expect("create local fs dir to store data");
+    config.local_fs_dir = t.path().to_str().unwrap().to_string();
+
```

Comment:
A tempdir guard needs to be held in `StoreTestContext`.
As the guard is dropped, the temp dir will be removed.

Similar to `meta_temp_dir`, here there should be another field holding the `local_fs_dir` guard.